process/vacuum.py

```diff
@@ -401,12 +401,10 @@ def vacuum(
             ceff[i] = 1.0e0 / (nduct / s[i] - 1.0e0 / (sp[i] * pumpn))
 
             #  Newton's method solution for duct diameter
-
             while True:
                 d[i] = 1.0e0
 
-                lap = 0
-                while True:
+                for _ in range(99):
```

Comment:
I think this should be 100.

Reply:
this one you may be right

Reply:
yeah this one should be 100 because of the placement of the break